tests/xtro-sharpie/EnumCheck.cs

```diff
@@ -239,8 +243,10 @@ public override void VisitEnumDecl (EnumDecl decl, VisitKind visitKind)
 							log = !managed_unsigned_values.ContainsKey (UInt64.MaxValue);
 							managed_unsigned_values.Remove (UInt64.MaxValue);
 						}
-						if (log)
-							Log.On (framework).Add ($"!missing-enum-value! {type.Name} native value {native_unsigned_values [value]} = {value} not bound");
+						if (log) {
+							if (!native_unsigned_values [value].Deprecated && !decl.IsDeprecated ())
```

Comment:
Do we need the nested if? Why not a single one like we had but with:
```suggestion
						if (log && !native_unsigned_values [value].Deprecated && !decl.IsDeprecated ())
```